File: client/go/model/BinaryObjectMetadata.go
package model

type BinaryObjectMetadata struct {
	ID               string     `json:"id"`
	Name             string     `json:"name"`
	Datagrams        []Datagram `json:"datagrams"`
	Sender           PClient    `json:"sender"`
	PusherInstanceId string     `json:"pusherInstanceId"`
}

File: verify/src/main/java/bmv/org/pushcaverifier/pushca/BinaryObjectMetadata.java
package bmv.org.pushcaverifier.pushca;

import bmv.org.pushcaverifier.client.PClient;
import java.util.List;

public record BinaryObjectMetadata(String id, String name, List<Datagram> datagrams, PClient sender,
                                   String pusherInstanceId) {

}
